apps/prairielearn/src/pages/partials/navTabs/instructorQuestion.ejs

```diff
@@ -21,4 +21,9 @@
     urlSuffix:      '/question/'+question.id+'/statistics',
     iconClasses:    'fas fa-chart-bar',
     tabLabel:       'Statistics'}) %>
+<%- include('navTab', {
+    activeSubPage:  'issues',
+    urlSuffix:      '/course_admin/issues?q=qid%3A'+question.qid,
```

Comment:
I think we need to take care to properly URL-encode the `qid` here.

What do you think about possibly showing a badge here with an issue count? That should already be available in `resLocals.open_issue_count` IIRC. Something like how GitHub displays issue/PR counts:

<img width="310" alt="Screenshot 2024-04-19 at 14 49 42" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/d74d7dd5-3681-4dc7-8cdf-9cb9f212aabe">

Reply:
Yes please, a red badge like we already use for the main Issues navbar item.

Reply:
Thanks to @nwalters512 help on Zoom, added a red badge in [381d728](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/381d7287d02a906cb1405663be90d4f84dc38348). 

![Screenshot 2024-04-24 at 12 10 40 PM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/b7f702f4-60a8-4d63-bfd8-37797cfed4e8)

Reply:
Perfect!